crates/modelardb_compression/src/lib.rs

```diff
@@ -976,7 +979,10 @@ pub mod test_util {
             }
             // Generates linear data.
             StructureOfValues::Linear => {
-                let a: i64 = thread_rng().gen_range(-10..10);
+                let mut a: i64 = 0;
+                while a == 0{
```

Comment:
Missing space between `0` and `{`, try reformatting the file with `rustfmt` and if that does not work add the space manually.

Reply:
Fixed.